### ISSUE TITLE
[Feature Request] Show better indication for inflow transfer or outflow transfer

### ISSUE THREAD

**Issue by ariccb** [actualbudget/actual#145]
is it possible to indicate as "in" or "out" or choose which direction these arrows point for transfers? For some reason my brain always associates going "right" as in, and "left" as out. That's the opposite of the default direction and makes things confusing for me. If there was a setting to choose the arrow direction, or to change from arrow to say "in" or "out" that'd be cool.
![image](https://user-images.githubusercontent.com/48956351/182409777-1a6e2469-efcf-404d-98f1-4185e623a00a.png)

**Comment by bpaulien** [comment]
Right, I'd interpret the first transaction shown above as going out of Tangerine Savings (because the arrow is pointing away from the account name) and the second one as in (because the arrow is pointing into the account name).

If the arrows were on the other (right) side, I'd interpret it the opposite way.  I'd think left pointing arrow would be in (because it's pointing towards the account name) and right would be out (pointing away from)
